task.sql

```diff
@@ -1 +1,38 @@
-# Write your SQL code for the database creation here. Good luck! 
+CREATE DATABASE ShopDB;
+
+USE ShopDB;
+
+CREATE TABLE Products (
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    Description VARCHAR(100),
+    Price DECIMAL(10, 2),
+    WarehouseAmount INT,
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Customers (
+    ID INT AUTO_INCREMENT,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Email VARCHAR(50),
+    Address VARCHAR(100),
+    PRIMARY KEY (ID)
+);
+
+CREATE TABLE Orders (
+    ID INT AUTO_INCREMENT,
+    CustomerID INT,
+    Date DATE,
+    PRIMARY KEY (ID),
+    FOREIGN KEY (CustomerID) REFERENCES Customers(ID) ON DELETE SET NULL
+);
+
+CREATE TABLE OrderItems (
+    ID INT AUTO_INCREMENT,
+    OrderID INT,
+    ProductID INT,
+    PRIMARY KEY (ID),
+    FOREIGN KEY (OrderID) REFERENCES Orders(ID) ON DELETE SET NULL,
+    FOREIGN KEY (ProductID) REFERENCES Products(ID) ON DELETE SET NULL
+);
```

Comment:
Add Eof(End-of-file), need to add a blank line at the end
![image](https://github.com/user-attachments/assets/518752de-6fc6-4761-9329-302a00dd8e46)